src/main/java/CinemaRoomRestService/RestService/api/dto/SeatDTO.java

```diff
@@ -0,0 +1,46 @@
+package CinemaRoomRestService.RestService.api.dto;
+
+public class SeatDTO {
+    private int row;
+    private int column;
+    private int price;
+
+    public int getRow() {
+        return row;
+    }
+
+    public int getColumn() {
+        return column;
+    }
+
+    public int getPrice() {
+        return price;
+    }
+
+    public static class Builder {
+        private SeatDTO seatDTO;
+
+        public Builder() {
+            this.seatDTO = new SeatDTO();
+        }
+
+        public Builder withRow(int row) {
```

Comment:
Currently when you understand Builder Pattern let's delete some boilerplate code and use library lombok for generating all stuff you need.

https://projectlombok.org/features/Builder
https://projectlombok.org/features/all
https://www.youtube.com/watch?v=QmsMWCIf3nc

At least I am waiting from you getters setters Builder. Replace them all.  You do not need to write them with annotation. Do not forget to add maven repository. Be aware with adding this annotations you have to understand what are they doing.

Reply:
Fixed